packages/flutter/test/widget/navigator_test.dart

```diff
@@ -40,6 +40,28 @@ class SecondComponentState extends State<SecondComponent> {
   }
 }
 
+class ThirdComponent extends StatelessComponent {
+  static final Key targetKey = new Key('foo');
+
+  final Function onException;
```

Comment:
Oh, I see this callback takes an argument. Use a typedef to define the signature.